application/.gitignore

```diff
@@ -1,6 +0,0 @@
-.vscode
```

Comment:
why are these no longer being ignored?

Reply:
I accidentally deleted the .gitignore in the applications folder.

I then put it back in using the .gitignore from the main branch's application

It should be working just as before?